packages/binding/src/transformer.rs

```diff
@@ -669,6 +663,54 @@ impl Transformer {
     }
   }
 
+  #[napi]
+  /// Support CSS3 color, e.g. rgba(255, 255, 255, .8)
+  pub fn from_svg(
+    input: Either<String, Buffer>,
+    background: Option<String>,
+  ) -> Result<Transformer> {
+    let mut fontdb = Database::new();
+    fontdb.load_system_fonts();
+    let options = Options::default();
+
+    let mut tree = match input {
+      Either::A(a) => usvg::Tree::from_str(a.as_str(), &options),
+      Either::B(b) => usvg::Tree::from_data(b.as_ref(), &options),
+    }
+    .map_err(|err| Error::from_reason(format!("{err}")))?;
+    tree.convert_text(&fontdb);
+
+    let mut size = tree.size.to_screen_size();
+    let min_svg_size = 1000;
+    while size.width() < min_svg_size || size.height() < min_svg_size {
```

Comment:
Is there a reason for this minimum size requirement? I'd prefer if this rendered my SVG at the input resolution